apps/lockfile-explorer-server/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+// This is a workaround for https://github.com/eslint/eslint/issues/3458
+require('@rushstack/eslint-config/patch/modern-module-resolution');
+
+module.exports = {
+  extends: ['@rushstack/eslint-config/profile/node'],
+  parserOptions: { tsconfigRootDir: __dirname },
+  rules: {
+    'max-lines': 'off'
```

Comment:
Can you suppress this for the specific file?

Reply:
FYI in order to suppress this rule, you have to use `/* eslint max-lines: off */`.  The other comment syntaxes (`// eslint-disable`) don't work because the rule is actually evaluated before the parser begins scanning the file.

Reply:
removed